2-Authorization-I/2-call-arm/SPA/src/components/DataDisplay.jsx

```diff
@@ -0,0 +1,89 @@
+import Table from 'react-bootstrap/Table';
+import { createClaimsTable } from '../utils/claimUtils';
+import { Container, Card, Row } from 'react-bootstrap';
+
+export const IdTokenData = (props) => {
+    const tokenClaims = createClaimsTable(props.idTokenClaims);
+
+    const tableRow = Object.keys(tokenClaims).map((key) => {
+        return (
+            <tr key={key}>
+                {tokenClaims[key].map((claimItem) => (
+                    <td key={claimItem} className="tableColumn">
```

Comment:
Can we get away with `key` values as non numerical values?

Reply:
It's recemented in React to use a unique value and not to use the index of an array.